README.md

```diff
@@ -280,6 +282,39 @@ auth = pusher.authenticate(
 # return `auth` as a response
 ```
 
+## End to End Encryption (Beta)
+
+This library supports end to end encryption of your private channels. This means that only you and your connected clients will be able to read your messages. Pusher cannot decrypt them. You can enable this feature by following these steps:
+
+1. You should first set up Private channels. This involves [creating an authentication endpoint on your server](https://pusher.com/docs/authenticating_users).
+
+2. Next, Specify your 32 character `encryption_master_key`. This is secret and you should never share this with anyone. Not even Pusher.
+
+```python
+
+import pusher
+
+pusher_client = pusher.Pusher(
+  app_id='yourappid',
+  key='yourkey',
+  secret='yoursecret',
+  encryption_master_key='XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
+  cluster='yourclustername',
+  ssl=True
+)
+
+pusher_client.trigger('private-encrypted-my-channel', 'my-event', {
+  'message': 'hello world'
+})
+```
+3. Channels where you wish to use end to end encryption should be prefixed with `private-encrypted-`.
```

Comment:
```suggestion
3. Channels where you wish to use end to end encryption must be prefixed with `private-encrypted-`.
```

Reply:
In the [public docs](https://pusher.com/docs/client_api_guide/client_encrypted_channels):
> Encrypted channels must be prefixed with `private-encrypted-`. See [channel naming conventions](https://pusher.com/docs/client_api_guide/client_channels#naming-channels).